app/src/main/res/values/strings.xml

```diff
@@ -1,28 +1,185 @@
-	<string
-		name="reset">Reset</string>
-	<string
-		name="clear_creds">Clear</string>
-	<string
-		name="cancel">Cancel</string>
-	<string
-		name="are_you_sure">Are you sure?</string>
-	<string
-		name="warning">Warning</string>
-	
-	<string
-		name="will_submit_stuff">Will submit listening information to this website
-	</string>
-	
-	<string
-		name="also_disable_np_title">Also disable now-playing</string>
-	<string
-		name="also_disable_np_summary">[Hmpf]</string>
+    <string name="ok">Ok</string>
+    <string name="close">Close</string>
+    <string name="remove">Remove</string>
+    <string name="reset">Reset</string>
+    <string name="clear_creds">Clear</string>
+    <string name="cancel">Cancel</string>
+    <string name="are_you_sure">Are you sure?</string>
+    <string name="warning">Warning</string>
+    <string name="will_submit_stuff">Will submit listening information to this website</string>
+    <string name="also_disable_np_title">Also disable now-playing</string>
+    <string name="also_disable_np_summary">[Hmpf]</string>
+    <string name="user_credentials_title">User credentials</string>
+    <string name="user_credentials_summary">Set username and password for your %1 account</string>
+    <string name="user_credentials_summary_no_napp">Set username and password for your account(s)</string>
+    <string name="user_credentials_enter_ok">Authenticate</string>
+    <string name="user_credentials_enter_cancel">Cancel</string>
+    <string name="user_credentials_clear_title">Clear credentials</string>
+    <string name="user_credentials_clear_summary">Clears username and session key</string>
+    <string name="supported_netapps_category_title">Supported websites</string>
+    <string name="username">Username</string>
+    <string name="password">Password</string>
+    <string name="auth_client_banned">This version of the app is deprecated, please upgrade</string>
+    <string name="auth_bad_auth">Wrong username/password</string>
+    <string name="auth_server_error">Server error: %1, retrying…</string>
+    <string name="auth_timing_error">Client side timing error, retrying…</string>
+    <string name="auth_network_error_retrying">Network error, retrying…</string>
+    <string name="auth_network_unfit">Not allowed to submit data on network, change in Options screen</string>
+    <string name="auth_updating">Authenticating…</string>
+    <string name="auth_internal_error">Internal error</string>
+    <string name="auth_just_error">Error, retrying…</string>
+    <string name="logged_in_just">Logged in as:</string>
+    <string name="logged_in_as">Logged in as: %1</string>
+    <string name="log_in_to">Sign in to</string>
+    <string name="not_logged_in">Not logged in</string>
+    <string name="create_user_title">Sign up</string>
+    <string name="create_user_summary">Sign up for an account at %1</string>
+    <string name="confirm_clear_creds">Are you sure you want to clear your user
+        credentials for %1? This
```

Comment:
These were originally like this. I'll correct to be singular, I guess they were for readability but not necessary  :+1: